site/learn/expert-advice/oauth/whats-new-with-oauth-2-1.md

```diff
@@ -0,0 +1,170 @@
+
+This grant is often used for native mobile applications. While this grant made it easier to migrate to OAuth with minimal application changes, it doesn't follow the typical delegation pattern. After all, the application server has full access to the credentials of the user, exactly what OAuth is designed to avoid. No longer can you leave the work of securing users' credentials and data to the OAuth server. With this grant, you must ensure that your application backend is just as secure.
+
+If you have a mobile application using this grant, you can either update the client to use an Authorization Code grant using PKCE or keep using your OAuth 2.0 compliant system.
+
+### No bearer tokens in the query string
+
+> Bearer token usage omits the use of bearer tokens in the query string of URIs as per Section 4.3.2 of [OAuth 2.0 Security Best Current Practices](https://tools.ietf.org/html/draft-ietf-oauth-security-topics-14)
+
+Bearer tokens, also known as access tokens, allow access to protected resources and therefore must be secured. They are called bearer tokens because merely possessing them gives you access. They're like a set of house keys. 
+
+These days, most tokens are JWTs. Clients store them securely and then use them to make API calls to the application server. The application server then uses the token to ensure the client calling the API has appropriate permissions. When first defined in [RFC 6750](https://tools.ietf.org/html/rfc6750), tokens were allowed in headers, POST bodies or query strings. The OAuth 2.1 draft prohibits sending a bearer token in a query string. 
+
+A query string and, more generally, any string in a URL, is never private. JavaScript executing on a page can access it. A URL or components thereof may be recorded in server log files, caches, or browser history. In general, if you want to pass information privately over the internet, don't use anything in a URL. Instead, use TLS and put the sensitive information in a POST body or HTTP header.
+
+### Limiting refresh tokens 
+
+> Refresh tokens must either be sender-constrained or one-time use as per Section 4.12.2 of [OAuth 2.0 Security Best Current Practices](https://tools.ietf.org/html/draft-ietf-oauth-security-topics-14)
+
+[Refresh tokens](https://tools.ietf.org/html/rfc6749#section-6) allow a client to retrieve new access tokens without reauthentication by the resource owner. If you need access to a resource for longer than an access token lives, or if you need infrequent access, refresh tokens can help. One example of where a refresh token might be used is when a user is logged into their email for weeks or months. Every time an access token expires, the client can present a refresh token and get a new access token.
```

Comment:
Can we link to https://fusionauth.io/docs/v1/tech/oauth/tokens/ in this section?

Reply:
Sure thing.